tensorflow/core/common_runtime/executor.cc

```diff
@@ -1791,6 +1794,29 @@ void ExecutorState::Process(TaggedNode tagged_node, int64 scheduled_nsec) {
     Entry* first_input = input_tensors + item.input_start;
     outputs.clear();
 
+    nvtxRangeId_t nvtx_range;
```

Comment:
Another case of adding too many lines that are not part of the main functionality of this function. Please refactor along the same lines suggested for eager/kernel_and_device.cc